sdk/core/src/services/anchoringHandler.ts

```diff
@@ -1,58 +1,68 @@
 import { ELEM_ANCHORED_DID_METHOD, ELEM_DID_METHOD, JOLO_DID_METHOD } from '../_defaultConfig'
 import { RegistryApiService } from '@affinidi/internal-api-clients'
-import { DidDocumentService, KeysService } from '@affinidi/common'
-
-const preprocessAnchoringParams = async (
-  api: RegistryApiService,
-  encryptedSeed: string,
-  password: string,
-  didDocument: any,
-  nonce: number,
-) => {
-  const keysService = new KeysService(encryptedSeed, password)
-  const { seed, didMethod: originalDidMethod } = keysService.decryptSeed()
-  const didMethod = originalDidMethod === ELEM_ANCHORED_DID_METHOD ? ELEM_DID_METHOD : originalDidMethod
-  if (didMethod === JOLO_DID_METHOD) {
-    const did = didDocument.id
-    const seedHex = seed.toString('hex')
-    const signedDidDocument = await keysService.signDidDocument(didDocument)
-
-    const { body: bodyDidDocument } = await api.putDocumentInIpfs({ document: signedDidDocument })
-    const didDocumentAddress = bodyDidDocument.hash
-
-    const {
-      body: { digestHex },
-    } = await api.createAnchorTransaction({ nonce, did, didDocumentAddress })
-
-    const transactionSignatureJson =
-      digestHex && digestHex !== '' ? await keysService.createTransactionSignature(digestHex, seedHex) : ''
-
-    const transactionPublicKey = KeysService.getAnchorTransactionPublicKey(seedHex, JOLO_DID_METHOD)
-    const ethereumPublicKeyHex = transactionPublicKey.toString('hex')
-
-    return { did, didDocumentAddress, ethereumPublicKeyHex, transactionSignatureJson, nonce }
-  }
+import { KeysService } from '@affinidi/common'
+import { DidMethod } from '../dto/shared.dto'
 
-  if (didMethod === ELEM_DID_METHOD) {
-    const didService = DidDocumentService.createDidDocumentService(keysService)
-    const did = didService.getMyDid()
+type AnchoringParams = {
+  registry: RegistryApiService
+  keysService: KeysService
+  didMethod: DidMethod
+  did: string
+  nonce: number
+  anchoredDidElem: boolean
+  additionalJoloParams?: {
+    didDocument: any
+    seedHex: string
+  }
+}
 
-    return { did, didDocumentAddress: '', ethereumPublicKeyHex: '', transactionSignatureJson: '' }
+const getPreparedJoloParams = async ({ registry, keysService, nonce, additionalJoloParams }: AnchoringParams) => {
```

Comment:
it is not only gets params but also make some processing around them, let's back what we consider in previous PR related to this https://github.com/affinityproject/affinidi-core-sdk/pull/150#discussion_r717397071

Reply:
Ultimately, it returns params used by `anchorDid` method of registry service.

I agree, `get` is not the best verb here.